requirements.txt

```diff
@@ -5,6 +5,12 @@ psycopg2==2.9.9 --no-binary=psycopg2
 Django==4.2.16
 libsass==0.23.0
 
+# Optional Trac dependencies that make DeprecationWarning go away.
+# When upgrading Trac or Python, check if these dependencies are still needed
+# by uninstalling them and running the test suite (or CI) with PYTHONWARNINGS=error
```

Comment:
I like this line a lot. I've never included it before. It's a good idea to explicitly state it.